apps-we-use/git.md

```diff
@@ -79,3 +79,12 @@ A few examples of excellent Pull Requests:
 
 In repositories where deployment involves database migrations and build verifications, we use a
 `PULL_REQUEST.md` template to provide delivery verification steps.
+
+
+### Keep Nitean in the loop
+
+It's important to regularly push your local changes to GitHub, even when they are only a work-in-progress. And post a short comment to the related GitHub issue(s). There are several benefits to doing so:
+* you have a backup of your work in case of a stupid mistake (whoooops I just deleted the whole folder) or a hardware failure,
```

Comment:
Lists need to be in sentence case

Reply:
Why? I would agree if a single list item is multiple sentences. But that is not the case here.

Reply:
To me this does not really read a list, rather a paragraph with bullet points inserted. So it's neither a list nor a paragraph and although the [Wikipedia style guide](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Lists#Bulleted_and_numbered_lists) does allow for it, I would prefer we stick to simply a list of sentences.